src/commands/docs/generateDocs.ts

```diff
@@ -13,11 +13,18 @@ import { getFoldersForDocsAllTargets } from './internal/getFoldersForDocsAllTarg
 
 export async function generateDocs(targetName: string, outDirectory: string) {
   const { doxyContent, buildDestinationDocsFolder } = getConstants()
-  if (!outDirectory) outDirectory = buildDestinationDocsFolder
 
   const config = await getLocalConfig()
-  const rootFolders = getFoldersForDocs(config)
 
+  if (!outDirectory)
+    outDirectory =
```

Comment:
This can be said more elegantly like this:
`outDirectory = config?.docConfig?.outDirectory ?? buildDestinationFolder`

Reply:
I have updated this assignment to:

```
    outDirectory = config?.docConfig?.outDirectory?.length
      ? config.docConfig.outDirectory
      : buildDestinationDocsFolder
```
We have placeholder by default: `"outDirectory": ""`

Reply:
`config?.docConfig?.outDirectory?.length` is basically the same as `config?.docConfig?.outDirectory`, because `""` is falsy.

Reply:
No, it's not. I faced this issue, then read the doc for Nullish coalescing operator

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator#using_the_nullish_coalescing_operator

Reply:
I'm pretty sure it is. I've just tried it out to confirm.
![image](https://user-images.githubusercontent.com/2980428/105831752-04de4580-5fbf-11eb-97f5-30a8febc860a.png)

Reply:
You'll only have issues when using nullish coalescing, because `""` is not `null`.

Reply:
Try with `??`
> outDirectory = config?.docConfig?.outDirectory ?? buildDestinationFolder
I will use `||` as per docs

Reply:
- [ ] `outDirectory = config?.docConfig?.outDirectory || buildDestinationDocsFolder`

Reply:
Yes, you can't use `??` effectively here because `outDirectory` can be `""`.
But you can use `config?.docConfig?.outDirectory || buildDestinationFolder`.

The point I'm trying to make is that `outDirectory?.length` is unnecessary.

Reply:
yep, `outDirectory?.length` is removed